components/mpas-ocean/src/analysis_members/Registry_time_series_stats_monthly_mean.xml

```diff
@@ -141,31 +142,45 @@
 			<var name="dThreshMLD"/>
 			<var name="normalVelocity"/>
 			<var name="vertVelocityTop"/>
+			<var name="zMid"/>
+			<var name="atmosphericPressure"/>
 			<var name="normalGMBolusVelocity"/>
 			<var name="vertGMBolusVelocityTop"/>
+			<var name="GMBolusVelocityZonal"/>
+			<var name="GMBolusVelocityMeridional"/>
+			<var name="cGMphaseSpeed"/>
+			<var name="velocityZonalTimesTemperature_GM"/>
+			<var name="velocityMeridionalTimesTemperature_GM"/>
+			<var name="normalGMBolusVelocityTimesTemperature"/>
+			<var name="normalGMBolusVelocitySquared"/>
+			<var name="gmKappaScaling"/>
+			<var name="gmBolusKappa"/>
+			<var name="mocStreamvalLatAndDepthGM"/>
+			<var name="mocStreamvalLatAndDepthRegionGM"/>
 			<var_struct name="tracersSurfaceFlux"/>
 			<var name="penetrativeTemperatureFlux"/>
 			<var name="latentHeatFlux"/>
 			<var name="sensibleHeatFlux"/>
 			<var name="longWaveHeatFluxUp"/>
 			<var name="longWaveHeatFluxDown"/>
 			<var name="seaIceHeatFlux"/>
-			<var name="icebergHeatFlux"/>
 			<var name="shortWaveHeatFlux"/>
+			<var name="frazilTemperatureTendency"/>
 			<var name="evaporationFlux"/>
 			<var name="seaIceSalinityFlux"/>
 			<var name="seaIceFreshWaterFlux"/>
-			<var name="icebergFreshWaterFlux"/>
 			<var name="riverRunoffFlux"/>
 			<var name="iceRunoffFlux"/>
 			<var name="rainFlux"/>
 			<var name="snowFlux"/>
-			<var_array name="vertNonLocalFlux"/>
+			<var name="vertNonLocalFlux"/>
```

Comment:
@darincomeau, this is a case where E3SM has it wrong and standalone MPAS-Ocean has it right:
https://github.com/E3SM-Project/E3SM/blob/5f6b107c8212c32ad6a07a43c9b42e516924a503/components/mpas-ocean/src/Registry.xml#L2856-L2859
Probably something to fix in E3SM in a different PR, but for this PR we should leave MPAS-Ocean as it is.

Reply:
thanks for catching that @xylar! change has been made.